support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -250,21 +254,26 @@ function WeeklyCheckoutForm(props: PropTypes) {
           }
           <FormSection title="When would you like your subscription to start?">
             <Rows>
-              <FieldsetWithError id="startDate" error={firstError('startDate', props.formErrors)} legend="When would you like your subscription to start?">
+              <RadioGroup
+                id="startDate"
+                error={firstError('startDate', props.formErrors)}
+                legend="When would you like your subscription to start?"
+              >
                 {days.map((day) => {
                   const [userDate, machineDate] = [formatUserDate(day), formatMachineDate(day)];
                   return (
-                    <RadioInput
-                      appearance="group"
-                      text={userDate}
+                    <Radio
+                      label={userDate}
+                      value={userDate}
                       name={machineDate}
                       checked={machineDate === props.startDate}
                       onChange={() => props.setStartDate(machineDate)}
+                      onClick={() => props.setStartDate(machineDate)}
```

Comment:
There's an `onChange` and `onClick` handler here, whilst on other `<Radio>` components we've just used `onChange`, is the `onClick` necessary?

Reply:
Click is not needed. Thanks George!